TwilioVerifySDK/TwilioVerify/Sources/Networking/NetworkProvider.swift

```diff
@@ -19,38 +19,186 @@
 
 import Foundation
 
-public typealias SuccessResponseBlock = (Response) -> ()
+// MARK: - Type aliases
+
+public typealias SuccessResponseBlock = (NetworkResponse) -> ()
 public typealias FailureBlock = (Error) -> ()
 
+// MARK: - Protocols
+
 public protocol NetworkProvider {
-  func execute(_ urlRequest: URLRequest, success: @escaping SuccessResponseBlock, failure: @escaping FailureBlock)
+  func execute(
+    _ urlRequest: URLRequest,
+    success: @escaping SuccessResponseBlock,
+    failure: @escaping FailureBlock
+  )
 }
 
-class NetworkAdapter: NetworkProvider {
-  
+// MARK: - Implementation
+
+final class NetworkAdapter: NetworkProvider {
```

Comment:
Can we move this to a new file? Including the URLSession extension

Reply:
Done on last commit